services-framework-proxy/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
 
     <uses-sdk
         android:minSdkVersion="10"
-        android:targetSdkVersion="23"/>
+        android:targetSdkVersion="33"/>
```

Comment:
I think it is better to remove minSdkVersion and targetSdkVersion from the xml since they are automatically added with the values from build.gradle